runtime/src/iree/base/internal/math.h

```diff
@@ -277,7 +277,13 @@ static inline float iree_math_f16_to_f32(const uint16_t f16_value) {
   const uint32_t sign = ((uint32_t)((f16_value & 0x8000u) >> 15)) << 31;
   uint32_t exp = ((f16_value & 0x7C00u) >> 10);
   uint32_t mantissa = 0;
-  if (exp > 0) {
+  if (exp == 0x1F) {
```

Comment:
Nit: `0x1Fu` to follow conventions used for other constants?

Similarly for the following.